sce.py

```diff
@@ -22,6 +22,8 @@
     help='SCE Service name')
 args = parser.parse_args()
 
+print(args)
```

Comment:
is there supposed to be a print statement here?

Reply:
I'm going to remove it